docs/docs/30-administration/10-server-config.md

```diff
@@ -17,8 +17,36 @@ services:
     environment:
       - [...]
 +     - WOODPECKER_OPEN=true
-+     - WOODPECKER_ORGS=dolores,dogpatch
+```
+
+When registration is closed, an administrator must add new users manually (with the CLI: `woodpecker-cli user add`) or by allowing specific users via `WOODPECKER_ADMIN` or `WOODPECKER_ORGS`.
+
+To close registration, but allow specific users:
+```diff
+# docker-compose.yml
+version: '3'
 
+services:
+  woodpecker-server:
+    [...]
+    environment:
+      - [...]
++     - WOODPECKER_OPEN=false
++     - WOODPECKER_ADMIN=johnsmith,janedoe
+```
+
+To close registration, but allow users who are members of approved organizations:
+```diff
+# docker-compose.yml
+version: '3'
+
+services:
+  woodpecker-server:
+    [...]
+    environment:
+      - [...]
++     - WOODPECKER_OPEN=false
++     - WOODPECKER_ORGS=dolores,dogpatch
```

Comment:
As the org option (afaik) is not related to open I would explain it without mentioning the open option

Reply:
You could allow only the org of your company. If `WOODPECKER_OPEN=false` is set an admin would be needed to add all users by hand. In case of `WOODPECKER_OPEN=true` the whole company could register on their own

Reply:
also WOODPECKER_OPEN has to be true if you wana use WOODPECKER_ORGS

https://github.com/woodpecker-ci/woodpecker/blob/a958049485d0284f48ed80e4650330ebe9b0aa65/server/api/login.go#L76-L91